src/common/type/pack.c

```diff
@@ -0,0 +1,1513 @@
+/***********************************************************************************************************************************
+Pack Type
+
+Each pack field begins with a one byte tag. The four high order bits of the tag contain the field type (PackType). The four lower
+order bits vary by type.
+
+Integer types (packTypeData[type].valueMultiBit) when the value is >= -1 and <= 1:
+  3 more value indicator set to 0
```

Comment:
Recommend using a dash and being consistent in using "bit" in the explanation. e.g. 
`3 - more value indicator bit set to 0`

Reply:
Updated in 919dc9668c93.